### PR TITLE
Highlight active page in nav

### LINKED ISSUE #418: Add active class on menu
For file: src/nyc_trees/apps/core/templates/base.html

Menu items should receive active class when a user is on that page.

```<li class="active"><a href="{% url 'group_list_page' %}">Groups</a></li>```

### PR DESCRIPTION
**Pull request by kdeloach**
Moves nav into a partial so we can highlight the active page in the
side nav.

Fixes #418.

### CODE REVIEW

File: src/nyc_trees/apps/core/templates/core/partials/nav.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                <li><a href="{{ event.get_user_checkin_url }}">Start Mapping</a></li>
+            {% endwith %}
+        {% endif %}
+        <li{% if active_page == "dashboard" %} class="active"{% endif %}><a href="{% url 'home_page' %}">Dashboard</a></li>
+        <li{% if active_page == "training" %} class="active"{% endif %}><a href="{% url 'training_summary_pure' %}">Training</a></li>
+        <li{% if active_page == "groups" %} class="active"{% endif %}><a href="{% url 'group_list_page' %}">Groups</a></li>
+        <li{% if active_page == "events" %} class="active"{% endif %}><a href="{% url 'events_list_page' %}">Events</a></li>
+        <li{% if active_page == "reservations" %} class="active"{% endif %}><a href="{% url 'reservations' %}">Reservations</a></li>
+        <li{% if active_page == "achievements" %} class="active"{% endif %}><a href="#">Achievements</a></li>
+        {% endif %}
```
Comment (maurizi) [inline src/nyc_trees/apps/core/templates/core/partials/nav.html]: It's a little hard to determine which `if` this `endif` belongs to.
Consider indenting the block inside the `{% if not soft_launch %}` to make it clearer

**Comment by maurizi** [comment]
+1

File: src/nyc_trees/apps/core/templates/core/partials/nav.html (hunk 1 above)
Comment (kdeloach) [inline src/nyc_trees/apps/core/templates/core/partials/nav.html]: Done.

Comment (jfrankl) [inline src/nyc_trees/apps/core/templates/core/partials/nav.html]: Looks good! +1

On Thu, Feb 5, 2015 at 2:04 PM, Kevin DeLoach <notifications@github.com>
wrote:

> In src/nyc_trees/apps/core/templates/core/partials/nav.html
> <https://github.com/azavea/nyc-trees/pull/457#discussion_r24188987>:
>
> > +        </span>
> > +    </div>
> > +    <ul class="nav-side-list">
> > +        {% if not soft_launch %}
> > +        {% if my_events_now|length == 1 %}
> > +            {% with event=my_events_now.0 %}
> > +                <li><a href="{{ event.get_user_checkin_url }}">Start Mapping</a></li>
> > +            {% endwith %}
> > +        {% endif %}
> > +        <li{% if active_page == "dashboard" %} class="active"{% endif %}><a href="{% url 'home_page' %}">Dashboard</a></li>
> > +        <li{% if active_page == "training" %} class="active"{% endif %}><a href="{% url 'training_summary_pure' %}">Training</a></li>
> > +        <li{% if active_page == "groups" %} class="active"{% endif %}><a href="{% url 'group_list_page' %}">Groups</a></li>
> > +        <li{% if active_page == "events" %} class="active"{% endif %}><a href="{% url 'events_list_page' %}">Events</a></li>
> > +        <li{% if active_page == "reservations" %} class="active"{% endif %}><a href="{% url 'reservations' %}">Reservations</a></li>
> > +        <li{% if active_page == "achievements" %} class="active"{% endif %}><a href="#">Achievements</a></li>
> > +        {% endif %}
>
> Done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/azavea/nyc-trees/pull/457/files#r24188987>.
>



-- 
Jeff Frankl, UI/UX Designer

Azavea  |  340 N 12th St, Suite 402, Philadelphia, PA
Web azavea.com  |  Blog azavea.com/blogs  |  Twitter @azavea
<http://twitter.com>

**Comment by kdeloach** [comment]
@azavea-bot rebuild

**Pull request merged**